src/Penrose/Functions.hs

```diff
@@ -1473,6 +1520,23 @@ contains [GPI rt@("Rectangle", _), GPI ar@("Arrow", _)] =
   in inRange startX lx rx + inRange startY ly ry + inRange endX lx rx +
      inRange endY ly ry
 
+contains [GPI sq@("Square", _), GPI ar@("Line", _)] =
```

Comment:
Why did we need these constraints again? Do they work well? I'm surprised that adding a bunch of `inRange` constraints would work

Reply:
I'm not sure if I was the one who wrote them. Checking the blame, it seems to be your change from 4 days ago?? https://github.com/penrose/penrose/blame/0eff99e63be0d75a3580f6cf505d5253aebac64a/src/Penrose/Functions.hs#L1520